api/src/main/resources/workbench.yaml

```diff
@@ -486,6 +486,37 @@ paths:
           schema:
             $ref: "#/definitions/ShareWorkspaceResponse"
 
+  /api/v1/workspaces/{workspaceNamespace}/{workspaceId}/clone:
+    post:
+      tags:
+        - workspaces
+      description: >
+        Clone an existing workspace, with optional modifications to workspace metadata. Caller will
```

Comment:
Clarifying "optional modification" -- does this just refer to the description field?

Reply:
Yeah, actually it's not really optional in that several fields are required. Reworded this.